backend/src/main/resources/graphql/main.graphqls

```
@@ -440,6 +440,116 @@ type FeatureFlag {
   value: Boolean!
 }
 
+input PatientReportInput {
+  firstName: String!
+  middleName: String
+  lastName: String!
+  suffix: String
+  email: String
+  phone: String
+  street: String
+  streetTwo: String
+  city: String
+  county: String
+  state: String
+  zipCode: String
+  country: String
+  sex: String
+  dateOfBirth: LocalDate!
+  race: String
+  ethnicity: String
+  tribalAffiliation: String
+}
+
+input ProviderReportInput {
+  firstName: String!
+  middleName: String
+  lastName: String!
+  suffix: String
+  npi: String!
+  street: String
+  streetTwo: String
+  city: String
+  county: String
+  state: String
+  zipCode: String
+  country: String
+  email: String
+  phone: String
+}
+
+input FacilityReportInput {
+  name: String!
+  phone: String
+  email: String
+  street: String
+  streetTwo: String
+  city: String
+  county: String
+  state: String
+  zipCode: String
+  country: String
+  clia: String!
+}
+
+input SpecimenInput {
+  snomedTypeCode: String!
```

Not sure if it matters, but it seems like the `snomedTypeCode` variable name should match what's in `SpecimenInput.java`, which is current `[typeSnomed](https://github.com/CDCgov/prime-simplereport/pull/8719/files/0991de5580bd83baa9f3d787b685b33fea9f026b#diff-135c9b93946605fd6f149907e4e195c01b844274ab7aed2eaf83a1cf3f1168d1R10)`. I'd lean towards changing the SpecimenInput.java field name instead of the name here.

---

Good catch, it looks like it causes a silent failure where `typeSnomed` is null on the `SpecimenInput` argument object of the mutation resolver. I agree we should update it to be `snomedTypeCode` and probably also add the `@NonNull` annotation on the SpecimenInput class. As we gather more requirements for what validation should be, we can add more of those annotations to the other input classes